lib/valkyrie/indexing/solr/indexing_adapter.rb

```diff
@@ -9,17 +9,13 @@ class IndexingAdapter
         ##
         # @!attribute [r] connection
         #   @return [RSolr::Client]
-        # @!attribute [r] resource_indexer
-        #   @return [Class]
         attr_reader :connection, :resource_indexer
 
         ##
         # @param connection [RSolr::Client] The RSolr connection to index to.
-        # @param resource_indexer [Class, #to_solr] An indexer which is able to
-        #   receive a `resource` argument and then has an instance method `#to_solr`
-        def initialize(connection: default_connection, resource_indexer: default_indexer)
+        def initialize(connection: default_connection)
           @connection = connection
-          @resource_indexer = resource_indexer
+          @resource_indexer = default_indexer
```

Comment:
looks like you either need to make the default `Hyrax::ValkyrieIndexer` or move the `.for` implementation somewhere that `NullIndexer` has a copy of the registry.

Reply:
That was sort of my question. I used `Hyrax::ValkyrieIndexer` as the default indexer here, but if we would prefer using the null indexer still, I can have it respond to the `.for` method and stub it to return the right indexer. 

That just depends on if we want this system as a whole to still utilize the null indexer or not.

Reply:
i think it's okay for us to use `Hyrax::Indexer` locally, and punt the `NullIndexer` question over to Valkyrie.